src/ansible/roles/ipa/tasks/main.yml

```diff
@@ -39,27 +39,39 @@
     ipa automountmap-del default auto.direct
   args:
     stdin: '{{ service.ipa.password }}'
+  register: automountmapdel
+  failed_when:
+    - 'automountmapdel.rc != 0 and "automount map not found" not in automountmapdel.stderr'
```

Comment:
Why?

Reply:
When running second time it is already deleted so "not found" is actually OK state.

Reply:
There were several other places where the playbook failed on second run because something was already done.